testnet/src/tests/integrations.rs

```diff
@@ -803,95 +666,3 @@ fn contract_stx_transfer() {
 
     run_loop.start(num_rounds);
 }
-
-#[test]
-fn bad_contract_tx_rollback() {
```

Comment:
This function seems like it may have been dropped due to a bad merge -- this test should be included.

Reply:
Good catch, thanks! This file has been a real stickler this week.
Just rebased my branch with master, it's fixed.

Reply:
Hmm -- the merge appears to still be problematic, I think that there's some lines that were added during Jude's mempool PR for the transaction submission, as well as some lines that were uncommented during the transaction rollback fix (#1415).